app/src/androidTest/java/net/skyscanner/backpack/skeleton/BpkBodyTextSkeletonTest.kt

```diff
@@ -0,0 +1,59 @@
+/**
+ * Backpack for Android - Skyscanner's Design System
+ *
+ * Copyright 2018 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package net.skyscanner.backpack.skeleton
+
+import androidx.test.annotation.UiThreadTest
+import androidx.test.ext.junit.runners.AndroidJUnit4
+import net.skyscanner.backpack.BpkSnapshotTest
+import org.junit.Before
+import org.junit.Test
+import org.junit.runner.RunWith
+
+@RunWith(AndroidJUnit4::class)
+class BpkBodyTextSkeletonTest : BpkSnapshotTest() {
+
+  @Before
+  fun setup() {
+    setDimensions(50, 400)
+  }
+
+  @Test
+  @UiThreadTest
+  fun screenshotTestSmallBodyTextSkeleton() {
```

Comment:
for the snapshot tests to work correctly you'll have to run them locally and commit the screenshots as well